### PR TITLE
LPS-184939 Add remaining localizable parameters in Image Picker

### PR DESCRIPTION
**Pull request by MirnaGama**
### **Steps to reproduce:**
1. From System Settings, enable Changeable Default Language option for Web Content. 
 2. Add a new structure 'test' with a text and image field.
 3. Create a new WC based on 'test' and define some values for the site's default locale (en_US):
 - Title: _title_
 - Text: _hello_
- Image: upload an image -- image1.
 4. Change the web content default language to es_ES in basic information.
 5. Define values for es_ES:
 - Title: _titulo_
 - Text: _hola_
 - Image: upload a different image -- image2.
 6. Change the language selector to ca-ES and observe the values set.

### _Expected result:_ 
Title: titulo; Text: hola Image: image2 ✅ 

### _Current result:_ 
Title: titulo; Text: hola; Image: image1 🚫

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ImagePicker/ImagePicker.es.js [hunk 1]
```diff
@@ -276,6 +280,15 @@ const Main = ({
...
 	...otherProps
 }) => {
+	const prevEditingLanguageId = usePrevious(editingLanguageId);
+
+	if (prevEditingLanguageId !== editingLanguageId && localizable) {
+		value =
+			localizedValue[editingLanguageId] !== undefined
+				? localizedValue[editingLanguageId]
+				: localizedValue[defaultLanguageId];
+	}
```
Comment (alinedoleron) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ImagePicker/ImagePicker.es.js:290]: Hi @MirnaGama, good job! 
Just one suggestion to simplify the code, we can use the _Nullish Coalescing_ operator in this case.

```suggestion
	if (prevEditingLanguageId !== editingLanguageId && localizable) {
		value = localizedValue[editingLanguageId] ?? localizedValue[defaultLanguageId];
	}
```

Thanks!

**Review by alinedoleron — CHANGES_REQUESTED** [review]
Hi @MirnaGama. Please, check my comment above. Thanks =D

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ImagePicker/ImagePicker.es.js (hunk 1 above)
Comment (MirnaGama) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ImagePicker/ImagePicker.es.js]: @alinedoleron Done ! :grin:

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by MirnaGama** [comment]
ci:test:sf

**Comment by MirnaGama** [comment]
ci:test:relevant

**Review by alinedoleron — APPROVED** [review]
LGTM!

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by alinedoleron** [comment]
ci:forward

**Comment by alinedoleron** [comment]
ci:forward:force

**Pull request closed without merging**